example/test_examples.py

```diff
@@ -22,7 +22,7 @@ def test_modelchain_example_flh(self):
         mc_e.calculate_power_output(weather, my_turbine, e126, dummy_turbine)
 
         assert_allclose(
-            2764.194772, (e126.power_output.sum() / e126.nominal_power), 0.01
+            2730.142, (e126.power_output.sum() / e126.nominal_power), 0.01
         )
```

Comment:
Are you sure that the new value is correct? Why is the old value wrong?

Reply:
Yes, it's because I changed the power_output_model to "power_coefficient_curve". And I changed the power output model because I wanted one example showing how to use the power coefficient curve. At an earlier point the power coefficient curve was used when calculating the feed-in for my_turbine2, but we now only provide its power curve.